softwarereview_intro.pt.Rmd

```diff
@@ -1,31 +1,31 @@
+A [coleção](https://ropensci.org/packages/) da rOpenSci é parcialmente resultado de contribuições de  membros(as) da equipe e parcialmente constituída por membros(as) da comunidade, o que significa que a coleção provém de uma grande diversidade de habilidades e de experiência de pessoas desenvolvedoras. Como garantir a qualidade de todo a coleção? É aí que entra a revisão por pares de software: os pacotes contribuídos pela comunidade passam por um processo de revisão transparente, construtivo, não adversarial e aberto. Para esse processo, que depende principalmente de trabalho voluntário, os(as) [editores(as) associados(as)](#associateditors) gerenciam o fluxo de submissões e garantem o andamento dos envios; os autores(as) criam, submetem e aprimoram seus pacotes; os [revisores(as)](#reviewers), duas pessoas por submissão, examinam o código e a experiência do usuário. [Esta publicação no blog](https://www.numfocus.org/blog/how-ropensci-uses-code-review-to-promote-reproducible-science/) escrito por editores(as) da rOpenSci é uma boa introdução à revisão por pares de software da rOpenSci.  Outros posts sobre a própria revisão e sobre pacotes revisados podem ser encontrados [através da tag "software-peer-review" no blog da rOpenSci](https://ropensci.org/tags/software-peer-review/).
 
-Você pode reconhecer os pacotes do rOpenSci que foram revisados por pares por meio de um emblema verde "peer-reviewed" em seu README, com links para suas revisões (cf [este exemplo](https://github.com/ropensci/restez#locally-query-genbank-)); e por meio de um ícone de comentário azul próximo à sua descrição em [página de pacotes do rOpenSci](https://ropensci.org/packages/) com links para as revisões.
+Você pode reconhecer os pacotes da rOpenSci que foram revisados por pares por meio de um selo (*badge*) verde com o texto "peer-reviewed" em seu README, com links para suas revisões ([por exemplo](https://github.com/ropensci/restez#locally-query-genbank-)); e por meio de um ícone de comentário azul próximo à sua descrição na [página de pacotes da rOpenSci](https://ropensci.org/packages/) com links para as revisões.
```

Comment:
It was quite difficult to understand what it meant in the translation.
I looked up on the website and this is an attempt to describe it a bit better.
But this is a change from the source book, I'm not sure how to proceed.

```suggestion
Você pode reconhecer os pacotes da rOpenSci que foram revisados por pares por meio de um selo (*badge*) verde com o texto "peer-reviewed" em seu README, com links para suas revisões ([por exemplo](https://github.com/ropensci/restez#locally-query-genbank-)); e por meio de um botão azul com o texto *"Peer-reviewed"* próximo à sua descrição na [página de pacotes da rOpenSci](https://ropensci.org/packages/) com links para as revisões.
```